### PR TITLE
ptpst.rst

### PR DESCRIPTION
**Pull request by meskarune**
Restructured text document to generate man page using rst2man

### CODE REVIEW

**Comment by kyrias** [comment]
Hmm, I thought I already made one, confused me.

Too bad that it doesn't render properly on GitHub, but oh well, not much that can be done about that.

File: doc/ptpst.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+:manpage:`libcurl(3)`
+
+See the documentation on pb, a lightweight pastebin at https://github.com/ptpb/pb/blob/master/pb/templates/index.rst
+
+Bugs
+----
+
+Report bugs for ptpst to https://github.com/HalosGhost/ptpst/issues
+
+Authors
```
Comment (kyrias) [inline doc/ptpst.rst]: Using an `AUTHORS` section is discourage these days, though when they exist I think it's more appropriate to have the author of the manpage instead.

Comment (kyrias) [inline doc/ptpst.rst]: (See eg `man-pages(7)`)

**Comment by HalosGhost** [comment]
On reading further, I would agree with @kyrias that the authors section should be removed. I'll handle all that when I merge these in and setup the rule for the docs to actually build.

**Pull request merged**